src/session/chat/mod.rs

```diff
@@ -137,7 +139,7 @@ mod imp {
                         "Last Message",
                         "The last message sent on this chat",
                         Message::static_type(),
-                        glib::ParamFlags::READWRITE,
+                        glib::ParamFlags::READWRITE | glib::ParamFlags::EXPLICIT_NOTIFY,
```

Comment:
I just noticed that we don't set `last-message` during construction. Maybe we should also consider doing this?